articles/virtual-desktop/app-attach-faq.md

```diff
@@ -55,6 +55,10 @@ Yes. MSIX app attach doesn't support auto-update for MSIX applications.
 
 All virtual machines (VMs) in a host pool that uses MSIX app attach must have read permissions on the file share where the MSIX images are stored. If it also uses Azure Files, they'll need to be granted both role-based access control (RBAC) and New Technology File System (NTFS) permissions.
 
+## Can I use Azure Active Directory Domain Services (AAD DS) with MSIX app attach?
+
+We do not support AAD DS with MSIX app attach, as the AAD DS computer objects are not synchronised to AAD DS and thus cannot be provided the RBAC permissions required in Azure Files
```

Comment:
```suggestion
MSIX app attach doesn't currently support Azure AD DS. Because Azure AD DS computer objects aren't synchronized to Azure AD DS, they can't get the required role-based access control (RBAC) permissions for Azure Files.
```

Reply:
This comment has it sync'ing to itself, but the other reworkding is good. Hence this should read:

MSIX app attach doesn't currently support Azure AD DS. Because Azure AD DS computer objects aren't synchronized to Azure Active Directory (AAD), they can't get the required role-based access control (RBAC) permissions for Azure Files.

Reply:
Azure AD DS is not the same as AD DS. We can't abbreviate "Azure" because it's not allowed by the style guide. "AAD" also must be changed to "Azure AD" because of branding requirements.

Reply:
So please set this to:

MSIX app attach doesn't currently support Azure AD DS. Because Azure AD DS computer objects aren't synchronized to Azure Active Directory (Azure AD), they can't get the required role-based access control (RBAC) permissions for Azure Files.